server/routes/handlers/projects.js

```diff
@@ -128,32 +128,48 @@ module.exports = {
     (req, res) => {
       const paginated = !(req.query.format && req.query.format === 'csv');
       req.query.scopes = { event_id: req.params.eventId };
-      return projectController
-        .getExtended(req.query, paginated)
-        .then((projects) => {
-          const dateFormat = date => new Date(date).toLocaleDateString();
-          if (!paginated) {
-            res.setHeader('Content-Type', 'text/csv');
-            const data = projects.toJSON();
-            const fields = [
-              { label: 'Name', value: 'name' },
-              { label: 'Description', value: 'description' },
-              { label: 'Category', value: 'category' },
-              { label: 'Supervisor Email', value: 'supervisor.email' },
-              { label: 'Owner Email', value: 'owner.email' },
-              { label: 'Created At', value: row => dateFormat(row.createdAt) },
-              { label: 'Updated At', value: row => dateFormat(row.updatedAt) },
+      return projectController.getExtended(req.query, paginated).then((projects) => {
+        const dateFormat = date => new Date(date).toLocaleDateString();
+        if (!paginated) {
+          res.setHeader('Content-Type', 'text/csv');
+          const data = projects.toJSON();
+          let fields = [
+            { label: 'Name', value: 'name' },
+            { label: 'Description', value: 'description' },
+            { label: 'Category', value: 'category' },
+            { label: 'Owner Email', value: 'owner.email' },
+            { label: 'Created At', value: row => dateFormat(row.createdAt) },
+            { label: 'Updated At', value: row => dateFormat(row.updatedAt) },
+            { label: 'Supervisor First Name', value: 'supervisor.firstName' },
+            { label: 'Supervisor Last Name', value: 'supervisor.lastName' },
+            { label: 'Supervisor Email', value: 'supervisor.email' },
+            { label: 'Supervisor Phone', value: 'supervisor.phone' },
+          ];
+
+          [0, 1, 2, 3, 4].forEach((i) => {
```

Comment:
The limitation to 5 people is soft (and used to be 4, 5 was for Pioneers), only the front-end restricts to that. I'd be tempted to do a Math.max() over the length of members instead. The CSV is not paginated, so even though it's expensive, it's probably easier (in term of work length) than doing a SQL max.
As another note, a ```fields.concat(new Array(memberLength).fill(0).map((val, index) => { return member; }))``` would be more to my taste, but that's nitpicking more than anything else

Reply:
Updated, thanks!